src/.vuepress/theme/components/Home.vue

```diff
@@ -24,8 +24,10 @@
 					title="Download latest release"
 					download
 				>
-					<font-awesome-icon icon="download" />
-					<span>Download {{ tagName || 'vX.X.X' }}</span>
+					<div class="icon-holder">
+						<MaterialIcon icon-name="get_app"/>
+						<span>Download {{ tagName || 'vX.X.X' }}</span>
```

Comment:
Maybe do this as it looks like less is changing on load, or have it set to latest release (`0.8.6`) as the updates are few, and if a new one comes it will be replace anyways.
```suggestion
						<span>Download {{ tagName || 'v0.0.0' }}</span>
```